scripts/cm_test_all_sandia

```diff
@@ -677,8 +677,8 @@ elif [ "$MACHINE" = "blake" ]; then
       # Format: (compiler module-list build-list exe-name warning-flag)
       # TODO: Failing toolchains:
       #"pgi/18.7.0 $BASE_MODULE_LIST $GCC_BUILD_LIST pgc++ $PGI_WARNING_FLAGS"
-    COMPILERS=("intel/18.1.163 $BASE_MODULE_LIST_INTEL $INTEL_BUILD_LIST icpc $INTEL_WARNING_FLAGS"
-               "gcc/7.2.0 $BASE_MODULE_LIST $GCC_BUILD_LIST g++ $GCC_WARNING_FLAGS"
+    COMPILERS=("intel/18.1.163 $BASE_MODULE_LIST_INTEL "OpenMP,Pthread" icpc $INTEL_WARNING_FLAGS"
+               "gcc/7.2.0 $BASE_MODULE_LIST "OpenMP_Serial" g++ $GCC_WARNING_FLAGS"
```

Comment:
This test is failing with the autotester. When I worked on this part of the script initially the blas and lapack were available on the system path, but it's possible that has since changed with system updates. If so, I can help tomorrow to update the script to use a module and set the cmake options to point to the libs etc.

Reply:
@e10harvey one more change after adding the openblas module to the module list, change this line (around 1079, will vary with your changes) from

https://github.com/kokkos/kokkos-kernels/blob/98e1c203f9e4d260013f3eea2beedf35ae6eed55/scripts/cm_test_all_sandia#L1079

to

```
if ([[ "$MACHINE" = white* ]] || [[ "$MACHINE" = weaver* ]] || [[ "$MACHINE" = blake* ]]) && [[ "$mod" = openblas* ]]; then
```

This will make sure the openblas tpl is found, as well as provide extra flags that are typically needed (-lgfortran, -lm) to prevent linkage errors when using blas not on a system path